NEW_README.md

```diff
@@ -0,0 +1,110 @@
+- Changes to the design which affect its compliance with the PRD MUST be re-approved by the individuals or groups specified in the original PRD.
+
+## Project Requirements Document (PRD)
+
+- A PRD MUST be created as a pull request.
+- A PRD MUST be a single document in a new directory in the `docs` directory.
+  - For example, if the project is called `cool-user-flow`, the PRD should be in `docs/cool-user-flow/prd.md`.
+  - A PRD MUST be written in markdown.
+- A PRD MUST identify:
+  - The project name
+  - The person responsible for the project
+  - Any individuals or groups that must approve the PRD.
+    - This MUST include
+      - someone on product
+      - an engineering team lead
+      - a devrel engineer if the project touches external stakeholders
+    - This MAY include
+      - someone from the legal team
+      - someone from the finance team
+      - someone from the sales team
+  - The PRD's target approval date
+  - The delivery deadline for the project
+  - The main user stories the project is intended to support
+  - How/If demand for those user stories has been measured
+  - Requirements describing the desired functionality/qualities in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) terms
+  - Each requirement SHOULD identify:
+    - Why the requirement exists
+    - Where the requirement comes from (e.g. what user is asking for it, or other observation inspired the requirement)
+    - The KPIs and target values that candidate solutions will be measured against
+  - Each requirement MAY:
+    - Give guidance on how the requirement may change beyond the delivery deadline
+- The PRD SHOULD:
+  - Define any key terms used in the requirements
+  - Include a list of assumptions made in creating the requirements
+  - Include a list of dependencies on other requirements
+  - Include commentary on where the ideal candidate solution should sit in a tradeoff space, to give guidance how different candidate solutions will be compared. For example, stating that between two candidate solutions, the one that has the lower "cost measured in X" will be preferred.
+- Once all required approvals have been received, the PRD SHOULD be merged within 72 hours.
+
+## Design Document (DD)
+
+- A DD MUST be created as a pull request.
+- A DD MUST be a single document in a new directory in the `docs` directory.
+  - For example, if the project is called `my-project`, the DD SHOULD be in `docs/my-project/dd.md`.
+  - A DD MUST be written in markdown.
+- A DD MUST identify:
+  - The PRD (including the commit hash) that the DD supports
+  - A title/name for the design
+  - The design approach
+  - The key architecture decisions
+  - How the design is expected to perform against the requirements in the PRD
+  - A lead engineer for the project
+  - A target completion date for the project
+  - Any individuals or groups that must approve the DD.
+    - This MUST include
+      - at least 2 engineers **other than the lead engineer**
```

Comment:
It's usual for the requirement to be "at least the TL of the project's team" plus at least one per external team affected.